src/main/resources/reference.conf

```diff
@@ -31,6 +31,35 @@ cassandra-journal {
   # Delay between connection retries
   connect-retry-delay = 5s
 
+  # Cassandra driver connection pool settings
+  # Documented at https://datastax.github.io/java-driver/features/pooling/
+  connection-pool {
+
+    # Create new connection threshold local
+    new-connection-threshold-local = 800
+
+    # Create new connection threshold remote
+    new-connection-threshold-remote = 200
+
+    # Connections per host core local
+    connections-per-host-core-local = 2
+
+    # CConnections per host max local
+    connections-per-host-max-local = 16
```

Comment:
I have decreased to 4. During my tests it was not common to reach the max-requests-per-connection threshold to have to increase connection count so I am assuming such value should be reasonable for many use cases.